pg2avro/pg2avro.py

```diff
@@ -184,16 +198,29 @@ def get_avro_row_dict(row, schema: Dict) -> Dict:
     :param row: Object to generate Avro row for.
     :type row: Object with compatible attributes, tuple, list or dict.
     :param schema: Schema to generate the Avro row with.
+    :param mapping_overrides: Custom mapping overrides.
     :return: Row dict.
     """
     avro_dict = {}
 
+    if mapping_overrides is None:
+        mapping_overrides = {}
+
     for schema_row in schema["fields"]:
         k = schema_row["name"]
         # TODO: check if row is compatible with schema.
         v = _get_row_attr(row, k, schema["fields"])
         column_type = schema_row["type"]
 
+        # Try to typecast if custom mapping and value are present.
+        if k in mapping_overrides and v is not None:
+            try:
+                v = mapping_overrides[k]["python_type"](v)
```

Comment:
This would fail if `python_type` is not valid - does not have casting function with the same name. The documentation mentions only 3 possible values, but they are not checked in the code.

Reply:
good point, and also those three are not even correct. Updated readme and check